restaurant/models.py

```diff
@@ -21,19 +21,20 @@ def __str__(self):
 		return self.name
 
 class Restaurant(models.Model):
-	owner = models.ForeignKey(get_user_model, on_delete=models.SET_NULL, null=True)
+	User = get_user_model()
+	owner = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
 	name = models.CharField(max_length=255)
 	email = models.EmailField()
 	phone = models.CharField(max_length=255)
-	delivery_fee = models.IntegerField()
-	average_delivery_time = models.FloatField()
-	minimum_order_value = models.IntegerField()
-	delivery_hours = models.JSONField(default=None)
+	delivery_fee = models.IntegerField(null=True)
+	average_delivery_time = models.FloatField(null=True)
+	minimum_order_value = models.IntegerField(null=True)
+	#delivery_hours = models.JSONField(default=None)
```

Comment:
delivery hours should be JSON field containing properties like
{
   delivery_start_time: 09:00am,
   delivery_end_time: 09:00pm
}